pkg/grafana/synthetic-monitoring-handler.go

```diff
@@ -67,20 +68,19 @@ func (h *SyntheticMonitoringHandler) newCheckResource(path string, filename stri
 	return resource
 }
 
-// Parse parses an interface{} object into a struct for this resource type
-func (h *SyntheticMonitoringHandler) Parse(path string, i interface{}) (grizzly.ResourceList, error) {
+// Parse parses a manifest object into a struct for this resource type
+func (h *SyntheticMonitoringHandler) Parse(m manifest.Manifest) (grizzly.ResourceList, error) {
 	resources := grizzly.ResourceList{}
-	msi := i.(map[string]interface{})
-	for k, v := range msi {
-		check := Check{}
-		err := mapstructure.Decode(v, &check)
-		if err != nil {
-			return nil, err
-		}
-		resource := h.newCheckResource(path, k, check)
-		key := resource.Key()
-		resources[key] = resource
+	spec := m["spec"].(map[string]interface{})
+	check := Check{}
```

Comment:
Beware that `Check` is a very generic word.

Reply:
Yeah, those types will disappear in future versions, replaced simply by `manifest.Manifest`.